### ISSUE TITLE
Surplus of QAFIBERSTATUS (red) fibers around z=0.95 on exp 200883, on 2023-10-14

### ISSUE THREAD

**Issue by dstndstn** [desihub/desisurveyops#153]
![tile-qa-4965-thru20231014](https://github.com/desihub/desisurveyops/assets/1065981/a228db95-bc22-4428-943c-86adac6e16a9)

**Comment by dstndstn** [comment]
Other exposures have mostly 40-60 fibers flagged, this exposure has 99, with many in a redshift band.  sframesky looks unremarkable.

**Comment by araichoor** [comment]
I think the reason is because this tile has been designed+observed a while ago, but the old exposure has efftime_spec=0.
so, if the stuck positioners have been moved in the meantime, they ll be rejected in the recent exposure.

i.e. I think it s ok (but I don t have time to double-check right now; a check would be to look if most of those red dots are stuck positioners).

I really need to add this info on the tile-qa, ie also report efftime_spec=0 exposures.

```
>>> d = Table.read("/global/cfs/cdirs/desi/spectro/redux/daily/exposures-daily.csv")
>>> d[d["TILEID"] == 4965]["TILEID", "NIGHT", "EXPID", "EFFTIME_SPEC"]
<Table length=2>
TILEID  NIGHT   EXPID  EFFTIME_SPEC
int64   int64   int64    float64   
------ -------- ------ ------------
  4965 20221030 150874          0.0
  4965 20231014 200883       1250.4
```

**Comment by dstndstn** [comment]
Makes sense!  Should I mark this tile as Yes in the tile qa? (and close this ticket)?

**Comment by schlafly** [comment]
Yes, @dstndstn , please go ahead and mark good and close.

**Comment by dstndstn** [comment]
Done!
